releasenotes/notes/improve-pep560-support-66f059006db8e3b2.yaml

```diff
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+    The :class:`PyDiGraph() <rustworkx.PyDiGraph>` and :class:`PyGraph() <rustworkx.PyGraph>`
+    classes now have better support for
+    `PEP 560 <https://peps.python.org/pep-0560/>`__. Building off of the previous
+    releases which introduced type annotations, the following code snippet is now valid:
+
+    .. jupyter-execute::
```

Comment:
Does this need to be executed? It seems like a static code-block since we just need syntax highlighting

Reply:
I have no preference either is fine. But I will say the advantage of executing is that it tests the code path.

 If you execute that with 0.15.1 you’ll get a runtime error!

Reply:
I'm fine leaving it. We're not super worried about our docs build time right now. If we were though this is the kind of thing we can change.